org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/ZigBeeBindingConstants.java

```diff
@@ -86,6 +86,10 @@
     public static final String CHANNEL_LABEL_FANCONTROL = "Fan Control";
     public static final ChannelTypeUID CHANNEL_FANCONTROL = new ChannelTypeUID("zigbee:fancontrol");
 
+    public static final String CHANNEL_NAME_BINARY_INPUT = "binary_input";
```

Comment:
Can I suggest we change this to just ```binaryswitch``` to keep it similar naming to other channels where we only have the underscore if it separates the cluster from the attribute (or something along those lines). Other channels such as ```activepower``` and ```batterylevel``` don't have this.